src/core/hle/service/http/http_c.cpp

```diff
@@ -55,11 +55,31 @@ const ResultCode ERROR_WRONG_CERT_HANDLE = // 0xD8A0A0C9
 const ResultCode ERROR_CERT_ALREADY_SET = // 0xD8A0A03D
     ResultCode(61, ErrorModule::HTTP, ErrorSummary::InvalidState, ErrorLevel::Permanent);
 
+static std::pair<std::string, std::string> SplitUrl(const std::string& url) {
```

Comment:
```suggestion
static std::pair<std::string_view, std::string_view> SplitUrl(std::string_view url) {
```
The function operates strictly on substrings of the input so it does not need to make copies